feature_selection/forward_selection.py

```diff
@@ -0,0 +1,79 @@
+def forward_selection(scorer, X, y, min_features=1, max_features=5):
```

Comment:
You can add the same code from other PRs to check inputs. I actually suggest creating a helper function to avoid redundancy for that purpose.

Reply:
Ok, sounds good idea. Thanks

Reply:
I will do it for milestone 3